languages/java/exercises/concept/case-statement/.docs/introduction.md

```diff
@@ -0,0 +1,23 @@
+Wikipedia describes a `switch` statement as "a type of selection control mechanism used to allow the value of a variable or expression to change the control flow of program".
+
+The mechanism involves the following keywords: `switch`, `case`, `break` and `default`.
+
+At their simplest they test a primitive or string expression and make a decision based on its value. For example:
```

Comment:
I know this is the original text in the C# exercise, but maybe you could come up with a way to describe this without paraphrasing wikipedia? I'm also not a big fan of the keyword listing. Maybe a little explanation could help there too?